Controller/CRUDController.php

```diff
@@ -53,7 +53,7 @@ class CRUDController extends Controller
      *
      * @return Response with json encoded data
      */
-    protected function renderJson($data, $status = 200, $headers = array(), Request $request = null)
+    protected function renderJson($data, $status = 200, $headers = array(), Request $request)
```

Comment:
Yeah that way we get less BC-breaks when changing the signature (no BC-break when adding an optional argument for instance)